plugins/obs-filters/chroma-key-filter.c

```diff
@@ -210,6 +221,7 @@ static void chroma_key_render(void *data, gs_effect_t *effect)
 	gs_effect_set_float(filter->similarity_param, filter->similarity);
 	gs_effect_set_float(filter->smoothness_param, filter->smoothness);
 	gs_effect_set_float(filter->spill_param, filter->spill);
+	gs_effect_set_float(filter->invert_param, filter->invert);
 
```

Comment:
type mismatch

Reply:
Oops, copy paste error. Let me patch that real quick.